manifests/base/routes.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     targetPort: 8443
   tls:
     insecureEdgeTerminationPolicy: Redirect
-    termination: Reencrypt
+    termination: reencrypt
```

Comment:
Wild that the OpenShift folks chose lowercase for this but not for all properties -- I would imagine all of these are just magic strings.

But it is in the API Explorer as such
![image](https://github.com/opendatahub-io/odh-dashboard/assets/8126518/6bd78287-4b34-4312-a2e1-693cb1a04948)